miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/DownloadedManifestCache.kt

```diff
@@ -127,13 +127,15 @@ internal class DownloadedManifestCache(context: Context) {
     }
 
     @VisibleForTesting
-    fun readFromCachedFile(miniAppId: String): CachedManifest? {
+    fun readFromCachedFile(miniAppId: String): CachedManifest? = try {
         val jsonToRead = File(getManifestPath(miniAppId), DEFAULT_FILE_NAME).bufferedReader()
             .use {
                 it.readText()
                     .dropLast(2) // for fixing the json format
             }
```

Comment:
I feel like the try catch to fix the file existence issue is kind of dirty. Should we create a `File` variable and test its existence, like

```suggestion
    fun readFromCachedFile(miniAppId: String): CachedManifest? {
        var file = File(getManifestPath(miniAppId), DEFAULT_FILE_NAME)
        if(!file.exists()) return null
        val jsonToRead = file.bufferedReader()
            .use {
                it.readText()
                    .dropLast(2) // for fixing the json format
            }
```
Maybe you can still keep the `try` for the json format errors

Reply:
Thanks for the suggestion, I have updated this part.